back/engines/commercial/aggressive_caching/app/controllers/aggressive_caching/patches/web_api/v1/admin_publications_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module AggressiveCaching
+  module Patches
+    module WebApi
+      module V1
+        module AdminPublicationsController
+          def self.included(base)
+            base.class_eval do
+              with_options if: :aggressive_caching_active? do
+                caches_action :index, expires_in: 1.minute, cache_path: -> { params.to_s }
```

Comment:
Wonder if `1.minute` could be a setting in the app_config for aggressive_caching? That way we could tune up and down depending on how it's looking

Reply:
Makes sense, will do

Reply:
In the end didn't do it, because the caches_action doesn't let me specify a proc for the expires_in. Since we don't have the context of the tenant when the cache action is defined, there is no elegant way to make it tenant-specific without touching the internals.